docs/lib/content/commands/npm-audit.md

```diff
@@ -88,7 +88,7 @@ The `sig` is generated using the following template: `${package.name}@${package.
 
 Keys response:
 
-- `expires`: null or a simplified extended [ISO 8601 format](https://en.wikipedia.org/wiki/ISO_8601"): `YYYY-MM-DDTHH:mm:ss.sssZ`
+- `expires`: null or a simplified extended [ISO 8601 format](https://en.wikipedia.org/wiki/ISO_8601): `YYYY-MM-DDTHH:mm:ss.sssZ`
 - `keydid`: sha256 fingerprint of the public key
 - `keytype`: only `ecdsa-sha2-nistp256` is currently supported by the npm CLI
 - `scheme`: only `ecdsa-sha2-nistp256` is currently supported by the npm CLI
```

Comment:
HTTPS://[PUBLIC]( KEY IDEAL API ENDPOINT)
THENEWFACTORY@ART&CONTENTS.RED 
Jcarlossantosferreiros77@gmail.com 

(Sheme:_only _ecdsa_sha_nistp256)
Currently by supported by the npm CLI